### PR TITLE
Handle routing errors

### PR DESCRIPTION
**Pull request by viesii**
part of

https://app.asana.com/0/1199937586489373/1202291573664006 and https://app.asana.com/0/1142794766483633/1201225530897124

### CODE REVIEW

File: app/controllers/errors_controller.rb [hunk 1]
```diff
@@ -6,6 +6,14 @@ class ErrorsController < ActionController::Base
...
   def not_found
     render status: :not_found
+
+    return unless request.referer
+
+    Rollbar.warning(
+      ["RoutingError: No route matches",
+       "[#{request.method}]",
+       "'#{request.original_url}'"].join(" ")
+    )
```
Comment (jeppester) [inline app/controllers/errors_controller.rb:16]: Do you know if `Rollbar.warning` automatically includes the stacktrace/request headers/etc.?

I think those will be very useful for eventually debugging dead links.

**Review by jeppester — APPROVED** [review]
I really like this, simple and easy to understand :+1:

File: app/controllers/errors_controller.rb (hunk 1 above)
Comment (jeppester) [reply app/controllers/errors_controller.rb:16]: After a quick investigation we've discovered that request headers ARE sent to rollbar. So we'll have all the info we need to debug dead links :partying_face:

Comment (MartinAbtion) [reply app/controllers/errors_controller.rb:16]: ah perfect! 💪 can you send a screenshot or something so i know where to look next time?

**Review by hcarreras — COMMENTED** [review]
Love it 👏

**Pull request merged**